glossary.yml

```
@@ -5580,6 +5592,12 @@
       A set of programs that are built, tested, and distributed as a collection so
       that they can run together.
 
+- slug: soure code
+  en:
+    term: "source code"
+    def: >
+      Is a human-readable text written in a specific programming language created by a programmer 
+      with the goal of setting exact rules for a computer-executable application.
```

```suggestion
      Human-readable text written in a programming language 
      with the goal of setting exact rules for a computer-executable application.
```

I don't agree with the

> goal of setting exact rules for

bit; that sounds more like it refers to parameter settings than source code as a whole. The concept of source code as a series of actions is probably more useful. Also, the definition should reference the difference between source code and compiled code explicitly; likely in a second sentence. Right now, this is only implied by the words 'human-readable'.